packages/jupyter-ai/jupyter_ai/chat_handlers/learn.py

```diff
@@ -67,17 +67,23 @@ def __init__(self, *args, **kwargs):
 
     def _load(self):
         """Loads the vector store."""
-        embeddings = self.get_embedding_model()
-        if not embeddings:
+        if self.index is not None:
             return
```

Comment:
Why is this needed? Is `_load` called anywhere else other than the `__init__` function?

Reply:
I agree it seems unnecessary, but I'm leaving it here to preserve the original logic until we can spend more time to verify. Specifically, I would like to remove this method in favor of `create()` anyways, since this method mostly does the same thing.